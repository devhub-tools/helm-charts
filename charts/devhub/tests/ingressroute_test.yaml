suite: IngressRoute
templates:
  - ingressroute.yaml
release:
  name: devhub
  namespace: devhub

tests:
  - it: should render ingressroute
    set:
      ingressRoute:
        enabled: true
    asserts:
      - matchSnapshot: {}

  - it: does not render ingressroute if enabled is false
    asserts:
      - hasDocuments:
          count: 0
