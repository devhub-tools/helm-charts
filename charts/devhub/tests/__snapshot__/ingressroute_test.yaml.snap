should render ingressroute:
  1: |
    apiVersion: traefik.io/v1alpha1
    kind: IngressRoute
    metadata:
      labels:
        app.kubernetes.io/instance: devhub
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: devhub
        app.kubernetes.io/version: v2.0.5
        helm.sh/chart: devhub-2.1.2
      name: devhub-http
    spec:
      entryPoints:
        - web
        - websecure
      routes:
        - kind: Rule
          match: Host(`devhub.example.com`)
          priority: 10
          services:
            - name: devhub
              passHostHeader: true
              port: http
      tls: {}
